tests/routines_test.py

```diff
@@ -14,13 +14,24 @@
 TIMEOUT = 360
 
 
+def set_trust(pubkey, seed, limit=None):
+    """Set trust limit for pubkey"""
+    LOGGER.info("setting trust limit: %s for address: %s", limit, pubkey)
+    prepared_transaction = routines.paket_stellar.prepare_trust(pubkey, limit)
+    routines.paket_stellar.submit_transaction_envelope(prepared_transaction, seed)
+
+
 class RoutinesTest(unittest.TestCase):
     """Test for routines"""
 
     def __init__(self, *args, **argv):
         super().__init__(*args, **argv)
-        self.eth_funder_seed = '4cdd30299b14203ba2289d6706acbf5e093fce6e170a48f3621c28d38f4ed20d'
-        self.eth_node = 'https://ropsten.infura.io/9S2cUwgCk4jYKYG85rxJ'
+        # pylint: disable=no-value-for-parameter
+        # pylint thinks web3-py insists on some useless kwargs here.
```

Comment:
@yaroslavr-itd , when having to disable pylint, I usually add comments to explain why.